### ISSUE TITLE
Support for coordinate and synthetic mask models (landmark_2d_106 , landmark_3d_68)

### ISSUE THREAD

**Issue by eous** [SthPhoenix/InsightFace-REST#125]
landmark_2d_106:
https://github.com/deepinsight/insightface/blob/master/alignment/coordinate_reg/image_infer.py#L9

landmark_3d_68:
https://github.com/deepinsight/insightface/blob/master/python-package/insightface/app/mask_renderer.py#L94

**Comment by SthPhoenix** [comment]
Hi! Main goal of this repo is face recognition. Those models make no improvements in terms of recognition - recognition module uses 5 key points for recognition, so 106 and 68 keypoints models are excessive.  
Also those models requires loose crops of faces for landmark detection, and uses non standard WxH input size, so those models could be added to pipeline, but at cost of huge performance loses.

**Comment by eous** [comment]
Understood, I was looking to do something along the lines of friendly detection which requires more points but it sounds like this project isn’t a great fit. Thank you for your response.

**Comment by SthPhoenix** [comment]
Yes, as I said before, it can be done, but will break performance. Though I'll keep in mind that this feature might be useful.
